lib/core/versions/latest/FeeManager.ts

```diff
@@ -49,6 +48,8 @@ export default class FeeManager {
       throw new SidetreeError(ErrorCode.OperationCountLessThanZero, `The number of operations: ${numberOfOperations} must be greater than 0`);
     }
 
+    // Requiring transaction fee prevents miner from paying lower fee because they get to decide what transactions to include in a block

```

Comment:
```suggestion
    // Requiring at least normalized fee prevents miner from paying lower fee because they get to decide what transactions to include in a block

```